Added cheatsheet description for short block (#2644)
cursorless-dev/cursorless: packages/cursorless-org-docs/src/docs/user/README.md

##### `"short block"`

> The `"short block"` scope is like `"block"`, but stops not only at empty lines but also stops if it would hit the nearest [surrounding pair](#surrounding-pair).

Reviewer: ```suggestion
The `"short block"` scope is like `"block"`, but in addition to stopping at empty lines, it will also stop before the nearest [surrounding pair](#surrounding-pair).
```

> The `"short block"` scope is like `"block"`, but in addition to stopping at empty lines, it will also stop before the nearest [surrounding pair](#surrounding-pair).

Reviewer: ```suggestion
The `"short block"` scope is like `"block"`, but in addition to stopping at empty lines, it will also stop at the boundary of a [surrounding pair](#surrounding-pair).
```

##### `"short paint"`

> The `"short paint"` scope is like `"paint"`, but stops not only at whitespace but also stops if it would hit the nearest [surrounding pair](#surrounding-pair). For example, given the text

Reviewer: ```suggestion
The `"short paint"` scope is like `"paint"`, but in addition to stopping at whitespace, it will also stop before the nearest [surrounding pair](#surrounding-pair). For example, given the text
```

> The `"short paint"` scope is like `"paint"`, but in addition to stopping at whitespace, it will also stop before the nearest [surrounding pair](#surrounding-pair). For example, given the text

Reviewer: ```suggestion
The `"short paint"` scope is like `"paint"`, but in addition to stopping at whitespace, it will also stop at the boundary of a [surrounding pair](#surrounding-pair). For example, given the text
```
